### PR TITLE
fix(curriculum): Consolidate challenge seed code comments

### PR DESCRIPTION
**Pull request by RandellDawson**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Related to PR #38040 

This PR is an attempt to consolidate/remove as many comments in the challenge seed code, to be able to create a small translation lookup object for translating the English comments to other languages.  It is a work in progress and I will be adding new commits over the next few days.

**Note:**  I plan to break this PR into smaller PRs because I removed comments from all of the Project Euler and Rosetta Code challenges.

I have narrowed the lookup table to the following unique comments:
```
// Only change code above this line
// Only change code below this line
// Change this line
// Example
// Setup
// Add your code above this line
// Add your code below this line
/* Add your code above this line */
/* Add your code below this line */
```
I still have approximately 110 comments to consolidate or remove.  See this Gist for comments still needing review](https://gist.github.com/RandellDawson/911b11d30615fb0987581d5945488db6).

### CODE REVIEW

File: curriculum/challenges/english/01-responsive-web-design/applied-visual-design/modify-fill-mode-of-an-animation.english.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ tests:
   button:hover {
     animation-name: background-color;
     animation-duration: 500ms;
-    /* add your code below this line */
+    /* Add your code below this line */
 
     /* add your code above this line */
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/01-responsive-web-design/applied-visual-design/modify-fill-mode-of-an-animation.english.md]: If I understood the gist correctly, this comment has already been consolidated.  Shouldn't it match the case of `/* Add your code below this line */`?

**Review by ojeytonwilliams — COMMENTED** [review]
I only noticed the one issue, but it was only a quick skim!

File: curriculum/challenges/english/01-responsive-web-design/applied-visual-design/modify-fill-mode-of-an-animation.english.md (hunk 1 above)
Comment (RandellDawson) [reply curriculum/challenges/english/01-responsive-web-design/applied-visual-design/modify-fill-mode-of-an-animation.english.md]: Yeah, for some reason, my script did not recognize it as the same comment.

**Comment by RandellDawson** [comment]
I am closing this PR, because I have created separate PRs to make it easier to review the changes.

**Pull request closed without merging**